core/dbt/adapters/sql/impl.py

```diff
@@ -63,32 +65,36 @@ def add_query(
                                           abridge_sql_log)
 
     @classmethod
-    def convert_text_type(cls, agate_table, col_idx):
+    def convert_text_type(cls, agate_table: agate.Table, col_idx: int) -> str:
         return "text"
 
     @classmethod
-    def convert_number_type(cls, agate_table, col_idx):
+    def convert_number_type(cls, agate_table: agate.Table, col_idx: int) -> str:
```

Comment:
Ugh, I always forget this - our pylint is set to 79 chars, not 80, so this line (just barely) fails it.
```suggestion
    def convert_number_type(
        cls, agate_table: agate.Table, col_idx: int
    ) -> str:
```

Reply:
Whoops, just pushed a fix.